### ISSUE TITLE
MyAsianTV down! Replace it with an alternative

### ISSUE THREAD

**Issue by Prudhvi-pln** [Prudhvi-pln/udb#25]
Current drama site - myasiantv.ac is down. Replace it with another site.

**Comment by Prudhvi-pln** [comment]
Found an alternative with huge catalog: https://asianbxkiun.pro/

**Comment by Prudhvi-pln** [comment]
Developed client for above but there are no updates after Nov 14th, 2024. Most of the sites are down!!!
Hunting for the diamond in dust now!

**Comment by Ichigo3766** [comment]
Hey found this and love the tool! Have you tried https://v2.dramacool.ba which has all the dramas? It would be really nice as the current one does not have new dramas or empty episodes. Lmk how things go!

**Comment by Prudhvi-pln** [comment]
Hi, thanks for the link. But unfortunately it doesn't have all the dramas (ex: what's wrong with secretary kim, trillion game).

**Comment by Ichigo3766** [comment]
I feel like one site will not contain everything but the one being used currently does not offer the latest dramas so would it be possible to make a client for the site like https://kissasian.com.lv just to pull the new popular dramas since they are updated? 

The current one does not even have new dramas like "Light Shop or squid game s2" whereas dramacool or kissasian does so would be nice to atleast have option for getting new ones. 

Lmk if it is suitable just to have variety.

**Comment by Prudhvi-pln** [comment]
I agree. We might need 2 sites, one having latest, other having the old but good catalog. Between I found a good site which has a better catalog, but it does not have multi resolution. I'll see if I can find any other and start the development.

**Comment by Prudhvi-pln** [comment]
Added a new client KissKH in v2.14.0. Along with support for MyAsianTV for the dramas till Nov 24, 2024.

Please create a new issue in case of any bugs/feature request.
